Fritz.Chatbot/Commands/AttentionCommand.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using Fritz.StreamLib.Core;
+using Microsoft.Extensions.Configuration;
+
+
+namespace Fritz.Chatbot.Commands
+{
+  public class AttentionCommand : IBasicCommand
+  {
+	private readonly IConfiguration Configuration;
+
+	public AttentionCommand(IConfiguration configuration)
+	{
+	  this.Configuration = configuration;
+	}
+
+	public string Trigger => "attention";
+
+	public string Description => "Play audio queue to divert attention to chat";
+
+	public TimeSpan? Cooldown => TimeSpan.FromSeconds(5);
```

Comment:
We should have a longer cooldown on this, or even make it configurable.  Let's start with a default of 10 minutes